Marlin/src/feature/spindle_laser.h

```diff
@@ -115,10 +111,16 @@ class SpindleLaser {
   FORCE_INLINE static void refresh() { apply_power(power); }
   FORCE_INLINE static void set_power(const uint8_t upwr) { power = upwr; refresh(); }
 
-  #if ENABLED(SPINDLE_LASER_PWM)
+  static inline void set_enabled(const bool enable) {
+    set_power(enable ? TERN(SPINDLE_LASER_PWM, (power ?: (unitPower ?: upower_to_ocr(cpwr_to_upwr(SPEED_POWER_STARTUP)))), 255) : 0);
```

Comment:
`unitPower` is specified in the cutter unit and not always a ocr value, therefore I think this should be

    set_power(enable ? TERN(SPINDLE_LASER_PWM, (power ?: upower_to_ocr(unitPower ?: cpwr_to_upwr(SPEED_POWER_STARTUP))), 255) : 0);

instead.